### DISCUSSION TITLE
How to not run the minecraft server at all

### DISCUSSION THREAD

**Discussion by ghost** [itzg/docker-minecraft-server#2758] [Q&A] [answered]
Hey, how would I be able to just not run the minecraft server and only use this image for the ability to easily acquire a minecraft server iamge

**Comment by itzg** [comment, marked as answer]
Sounds like this would help https://docker-minecraft-server.readthedocs.io/en/latest/configuration/misc-options/#setup-only

**Reply by ghost** [reply]
Thank you so much!
